Articles/Blog/2020-03-maintainers-drive-sustainability.md

```diff
@@ -8,60 +8,60 @@
+**Limited software development practices.**  The experiences of ParFlow’s development team match an [earlier BSSw blog on continuous technology refreshment](https://bssw.io/blog_posts/continuous-technology-refreshment-an-introduction-using-recent-tech-refresh-experiences-on-visit) (CTR), which highlights a core issue: Applications that expect to continue to work over time cannot be static but must undergo "*periodic upgrade or replacement of infrastructure to deliver continued reliability, improved speed, capacity, and/or new features*" to respond to the developing needs of users and workflows.  The ParFlow team had not adopted formal software engineering practices including CTR because of the lack of funding targeted at software maintenance.
 
-**New capabilities prioritized over good software development.** In addition to necessary ongoing maintenance, most research codes are never ‘done’.    Even when there wasn't funding targeted for ParFlow development, new capabilities were being created by projects using ParFlow but the development was done in an ad-hoc manner.  Lack of software development processes lead to the code base becoming splintered across organizations, no one bothered to run the existing regression tests,  bugs went unfixed, and users found conflicting hacks for building on current operating systems.   
+**New capabilities prioritized over good software development.** In addition to necessary ongoing maintenance, most research codes are never "done."    Even when there wasn't funding targeted for ParFlow development, new capabilities were being created by projects using ParFlow; but the development was done in an ad hoc manner.  Lack of software development processes led to the code base becoming splintered across organizations; no one bothered to run the existing regression tests,  bugs went unfixed, and users found conflicting hacks for building on current operating systems.   
 
-**Intermittent funding for development and maintenance of the software.** A key lesson that we have made out of observing the development and evolution of ParFlow has been the need to have one or more identified maintainers to keep the software in a running and usable state.   Looking back at the on/off cycle of funding cycle we have observed several recurring patterns of failure when there are no maintainers.   
+**Intermittent funding for development and maintenance of the software.** A key lesson that we have learned from observing the development and evolution of ParFlow has been the need to have one or more identified maintainers to keep the software in a running and usable state.   Looking back at the on/off cycle of funding cycle, we have observed several recurring patterns of failure when there are no maintainers.   
```

Comment:
"cycle of funding cycle" should be cycle of funding.

Reply:
or just "funding cycle"

Reply:
Done